.github/actions/setup-pnpm/action.yml

```
@@ -0,0 +1,23 @@
+name: setup-pnpm
+description: "This is a composite GitHub Action that checks out the repo, sets up pnpm, node and installs the project's dependencies."
```

nits: It seems that this action doesn't include checkout step
```suggestion
description: "This is a composite GitHub Action that sets up pnpm, node and installs the project's dependencies."
```

---

Thanks!!